RadixWallet/Clients/ProfileStore/ProfileStore+AsyncSequence+Updates.swift

```diff
@@ -0,0 +1,66 @@
+
+
+// MARK: Public
+extension ProfileStore {
+	/// The current network if any
+	public func network() async throws -> Profile.Network {
+		try profile.network(id: profile.networkID)
```

Comment:
This looks like it would be more natural on profile itself?

Reply:
Why is it async by the way, old habits?

Reply:
fixed